### PR TITLE
Front is works, needs styling and navigation

### PR DESCRIPTION
**Pull request by kentstone84**
We have basic navigation and have started building functionality into drop down menus and page access. Pushing up to focus on EOD tasks and resume.

### CODE REVIEW

File: pig-tinder-frontend/src/App.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+   <>
+ <Header />
```
Comment (WalkerTexas-coder) [inline pig-tinder-frontend/src/App.js:24]: indent header twice

File: pig-tinder-frontend/src/Pages/NotFound.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const NotFound = () => {
+  return (
+    <div>NotFound</div>
+  )
+}
+
```
Comment (WalkerTexas-coder) [inline pig-tinder-frontend/src/Pages/NotFound.js:8]: Notfound needs styles

File: pig-tinder-frontend/src/Components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div></div>
+  )
+}
+
+export default Footer
```
Comment (WalkerTexas-coder) [inline pig-tinder-frontend/src/Components/Footer.js:9]: Footer needs styles

**Review by WalkerTexas-coder — APPROVED** [review]
Needs styles for Footer and Not found. Formatting for code comment below as well.

File: pig-tinder-frontend/src/Components/Footer.js (hunk 3 above)
Comment (kentstone84) [reply pig-tinder-frontend/src/Components/Footer.js:9]: Yes, as i said in comments we pushed to fix resumes and sent to paul. Will fix styling and functionality at a later time. Just needed to push what has been completed.

**Pull request merged**